app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -12,8 +12,12 @@ import org.oppia.app.settings.profile.ProfileListActivity
 import javax.inject.Inject
 
 /** Activity for Administrator Controls. */
-class AdministratorControlsActivity : InjectableAppCompatActivity(), RouteToProfileListListener, RouteToAppVersionListener {
-  @Inject lateinit var administratorControlsActivityPresenter: AdministratorControlsActivityPresenter
+class AdministratorControlsActivity :
+  InjectableAppCompatActivity(),
+  RouteToProfileListListener,
+  RouteToAppVersionListener {
```

Comment:
Is this the default behaviour of ktlint?
Meaning I was expecting it to be like this:
```
class AdministratorControlsActivity : InjectableAppCompatActivity(),
  RouteToProfileListListener,
  RouteToAppVersionListener {
```

Reply:
Ktlint says there should be a new line after `:` and if there are multiple things there should be new line after `,`

Reply:
Okay than in that case, it should be applied to https://github.com/oppia/oppia-android/pull/1291#discussion_r439205087 for better consistency.

Reply:
yeah, definitely